keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -127,6 +135,55 @@ The mitigations to those risks:
 
 ## Design Details
 
+### Following sections will be added
+| Field   | Comments                                            |
+|---------|-----------------------------------------------------|
+| Linux   | new section for linux (platform-specific) options   |
+| Windows | new section for windows (platform-specific) options |
+
+### Following fields will be moved (without any change in name, data-type and default values)
+| v1alpha1               | v1alpha2            | Comments                                                         |
+|------------------------|---------------------|------------------------------------------------------------------|
+| Conntrack              | Linux.Conntrack     | moved from root(generic) to linux (platform-specific) section    |
+| OOMScoreAdj            | Linux.OOMScoreAdj   | moved from root(generic) to linux (platform-specific) section    |
+| IPTables.MasqueradeAll | Linux.MasqueradeAll | moved from iptables (backend-specific) to root (generic) section |
+| NFTables.MasqueradeAll | Linux.MasqueradeAll | moved from nftables (backend-specific) to root (generic) section |
+| IPTables.SyncPeriod    | SyncPeriod          | moved from iptables (backend-specific) to root (generic) section |
+| NFTables.SyncPeriod    | SyncPeriod          | moved from nftables (backend-specific) to root (generic) section |
+| IPVS.SyncPeriod        | SyncPeriod          | moved from ipvs (backend-specific) to to root (generic) section  |
+| IPTables.MinSyncPeriod | MinSyncPeriod       | moved from iptables (backend-specific) to root (generic) section |
+| NFTables.MinSyncPeriod | MinSyncPeriod       | moved from nftables (backend-specific) to root (generic) section |
+| IPVS.MinSyncPeriod     | MinSyncPeriod       | moved from ipvs (backend-specific) to root (generic) section     |
+
+### Following fields will be changed
+| v1alpha1           | v1alpha2                 | DataType     | Comments                                                                                                       |
+|--------------------|--------------------------|--------------|----------------------------------------------------------------------------------------------------------------|
+| ClusterCIDR        | DetectLocal.ClusterCIDRs | list[string] | list of CIDR ranges for detecting local traffic                                                                |
+| BindAddress        | NodeIPOverride           | <same>       | no change in data type                                                                                         |
```

Comment:
actually, let's make NodeIPOverride a list[string] too, so you can actually properly override dual-stack node IPs.